bin/yangIetf.py

```diff
@@ -152,11 +145,12 @@ def custom_print(message: str):
                         help='Optional flag that determines wheter compilation should be run '
                              'for all files even if they have not been changed '
                              'or even if the validators versions have not been changed.',
-                        type=bool,
-                        default=False)
+                        action='store_true')
```

Comment:
`default=False` should stay as original

Reply:
`default=False` is implicit in `action='store_true'`, but I can add it back to make the formatting more similar to other options.

Reply:
Ah Okay, I thought the default was True, because yangIetf.py started parsing all - even archived drafts for some reason.

Reply:
That was because I accidentally overrode the default with `default='{}/my-id-mirror/'.format(ietf_directory)`. Fixed now.